### DISCUSSION TITLE
Welcome to gmsh2exo Discussions!

### DISCUSSION THREAD

**Discussion by andrsd** [andrsd/gmsh2exo#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fab4100** [comment]
The code of your project has been very helpful to me! I did a little diggin few months back and I found only this converter to be actually useful.  I appreciate your efforts!

Best,
Fabs

**Reply by andrsd** [reply]
Thanks for stopping by and dropping a line. Glad to hear this little tool is useful :+1:
